app/plugins/newgrounds.ts

```diff
@@ -31,7 +31,16 @@ export class NewgroundsPlugin extends BaseSitePlugin {
     }
 
     getCreator() {
-        return querySelector(".item-user .item-details-main")?.textContent.trim();
+        // If there are multiple creators listed, concatenate all their
+        // names together, with commas between.
+        let creatorNameElts = querySelectorAll(".item-user .item-details-main");
+        let creatorNames = [];
+        for (let name of creatorNameElts) {
+            creatorNames.push(name.textContent.trim());
+        }
+        let creators = creatorNames.join(',');
```

Comment:
Obviously, this is fine, though idiomatic Javascript for this would likely be something like
```typescript
const creatorNames = creatorNameElts.map(name => name.textContent.trim());
const creators = creatorNames.join(',');
```

Reply:
I will write Perl in any language.  :D  Will fix.